roles/importer/files/importer/fwo_api_oo.py

```diff
@@ -2,28 +2,33 @@
 import requests
 import json
 import traceback
-import copy
 import time
 
 import fwo_globals
+
 from fwo_log import getFwoLogger
-from fwo_const import fwo_api_http_import_timeout, api_call_chunk_size
+from fwo_const import fwo_api_http_import_timeout
 from fwo_exceptions import FwoApiServiceUnavailable, FwoApiTimeout
-from fwo_base import ChunkableVariableType
+from roles.importer.files.importer.query_analyzer import QueryAnalyzer
```

Comment:
always specify paths relative to importer dir:

        from query_analyzer import QueryAnalyzer